src/relay/op/tensor/transform.cc

```diff
@@ -1269,6 +1269,93 @@ RELAY_REGISTER_OP("repeat")
     .set_attr<FTVMCompute>("FTVMCompute", RepeatCompute)
     .set_attr<TOpPattern>("TOpPattern", kBroadcast);
 
+// meshgrid operator
+TVM_REGISTER_NODE_TYPE(MeshgridAttrs);
```

Comment:
we also need to add it in https://github.com/apache/incubator-tvm/blob/master/python/tvm/relay/op/op_attrs.py

Reply:
Thanks, added.